### DISCUSSION TITLE
Linux build machine specification stack update to support Android 14

### DISCUSSION THREAD

**Discussion by szymonequiqo** [codemagic-ci-cd/codemagic-docs#2773] [Ideas]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Check Linux build machine specification:
https://docs.codemagic.io/specs/versions-linux/

### Expected results

There is a build machine with the latest, most commonly used tools ready to run e2e test with the most popular emulators.

It is possible to run e2e tests on Android emulator with API 34(Android14)

### Actual results

Linux build machine has very a old tech stack -> https://docs.codemagic.io/specs/versions-linux/
- 2 years old Android Studio 2022.3
- emulators with Android 11, where Android 14 is the latest, and the most used Android versions for our customers.

**Comment by helinanever** [comment]
Hi @szymonequiqo , sorry for the long delay on getting back to you. We have just updated the Linux machines and installed the Android 14 emulator, see the [announcement](https://github.com/orgs/codemagic-ci-cd/discussions/2832) for more information.

**Comment by szymonequiqo** [comment]
@helinanever thank you :)
